test/BEST_PRACTICES.md

```diff
@@ -0,0 +1,53 @@
+# Test Best Practices
+
+This is a collection of best practices to write high quality tests. The best practices
+can be referenced in review comments to speed up the review process.
```

Comment:
Suggested wording:
'...These best practices should be referenced in the comments that you provide when you perform code reviews of tests...'

Reply:
@ldimaggi if you want to make the suggestion even more effectively you can use the github suggestions so that I can accept them with one click, less work for me :)